### PR TITLE
404 task create dev deployment

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

Please include a summary of the changes and the related issue.

Resolves #(issue)

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
LGTM

File: kubernetes/base/keycloak/keycloak-service.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ metadata:
 spec:
   ports:
     - name: keycloak-svc-port
-      port: 8081
+      port: 8080
```
Comment (bigglesandginger) [inline kubernetes/base/keycloak/keycloak-service.yaml:13]: Won't this force changes to the ingress?

Comment (bigglesandginger) [reply kubernetes/base/keycloak/keycloak-service.yaml:13]: Or is this only used when running locally (which we don't do currently) ???

Comment (dgauldie) [reply kubernetes/base/keycloak/keycloak-service.yaml:13]: the ingress points to 443 anyway. so this does not affect ingress

**Pull request merged**